common/test/acceptance/tests/studio/test_studio_home.py

```diff
@@ -173,3 +178,78 @@ def test_studio_language_change(self):
             get_selected_option_text(language_selector),
             u'Dummy Language (Esperanto)'
         )
+
+
+class CourseNotEnrollTest(WebAppTest):
+    """
+    Test that we can create a new content library on the studio home page.
+    """
+
+    def setUp(self):
+        """
+        Load the helper for the home page (dashboard page)
+        """
+        super(CourseNotEnrollTest, self).setUp()
+
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.course_name = "New Course Name"
+        self.course_org = "orgX"
+        self.course_number = str(uuid.uuid4().get_hex().upper()[0:6])
+        self.course_run = "2015_T2"
+
+    def course_id(self):
+        """
+        Returns the serialized course_key for the test
+        """
+        # TODO - is there a better way to make this agnostic to the underlying default module store?
+        default_store = os.environ.get('DEFAULT_STORE', 'draft')
+        course_key = CourseLocator(
+            self.course_org,
+            self.course_number,
+            self.course_run,
+            deprecated=(default_store == 'draft')
+        )
+        return unicode(course_key)
+
+    def test_unenroll_course(self):
+        """
+        From the home page:
```

Comment:
I'm not sure why this is a studio test — you should be able to do all this from the LMS.

Let's just test the following two workflows:
1)
Given I am global staff
Given I am not enrolled in course X
When I go to the courseware index view of course X
Then I am redirected to the EnrollStaff page
When I click "enroll",
Then I am enrolled in the course
And I am redirected to the courseware.

2)
Given I am global staff
Given I am not enrolled in course X
When I go to a "jump_to" view in course X
Then I am redirected to the EnrollStaff page
When I click "enroll",
Then I am enrolled in the course

We should also test the cases when you click "don't enroll"

In fact, now that I think of it, we can test this whole workflow in unittests, very similarly to how you already did, below. So bok choy tests may not even be necessary at all here.